M2/Macaulay2/packages/Macaulay2Doc/functions/package-doc.m2

```diff
@@ -271,6 +271,7 @@ Node
     [newPackage, PackageExports]
     [newPackage, PackageImports]
     [newPackage, Reload]
+    [newPackage, TestFiles]
```

Comment:
This is complicating the testing system rather than simplifying it, and I would personally rather not add yet another option to `newPackage`.

Reply:
Fair enough -- I wasn't that excited about this idea either.

Another possibility would be to change `currentLayout#"packagetests"` to point to a subdirectory of the auxiliary files directory (say `test` or `tests`) instead of the current behavior, where it's a subdirectory of the package's documentation directory.  Then any files in that directory would automatically get loaded as tests, as currently happens for `Core`.

Reply:
`Polyhedra/tests` might get in the way, but in principle that's fine with me.